src/components/heading/heading.component.js

```diff
@@ -168,6 +174,20 @@ class Heading extends React.Component {
     return this.props.divider ? <StyledDivider data-element="divider" /> : null;
   }
 
+  /**
+   * Returns pills if provided
+   *
+   * @method pills
+   * @return {Object} JSX
+   */
+  get pills() {
+    return this.props.pills ? (
+      <StyledHeadingPills data-element="pills">
+        {this.props.pills}
+      </StyledHeadingPills>
+    ) : null;
+  }
```

Comment:
I think the same but figured out that actual difference would be the word get and ().
Decided that it will be slightly quicker for someone to understand whats going on before the refactor if everything is the same.
OFC i can just quickly refactor the whole file, but thought the diff would be unnecessarily big ;)